includes/polling/applications/memcached.inc.php

```diff
@@ -1,6 +1,13 @@
 <?php
 
-$data = $agent_data['app']['memcached'][$app['app_instance']];
+if (!empty($agent_data['app']['memcached'])) {
+    $data = $agent_data['app']['memcached'][$app['app_instance']];
+} else {
+    $options = '-O qv';
+    $oid     = '.1.3.6.1.4.1.8072.1.3.2.3.1.2.9.109.101.109.99.97.99.104.101.100';
+    $result  = snmp_get($device, $oid, $options);
+    $data    = reset(unserialize(str_replace("<<<app-memcached>>>\n", '', $result)));
```

Comment:
Scrutinizer bug, reset() expects a reference.
So you might as well set it to a variable and then use `[0]`